canvas_sdk/effects/protocol_card/recommendation.py

```diff
@@ -0,0 +1,28 @@
+from typing import Literal
+
+from pydantic import BaseModel, ConfigDict
+
+
+class Recommendation(BaseModel):
```

Comment:
This could probably live in the same file as ProtocolCard, unless there is some benefit to keeping them separate.

Reply:
will do!